pkg/api/controller.go

```diff
@@ -527,16 +527,26 @@ func CreateRepoDBDriver(storageConfig config.StorageConfig, log log.Logger) (rep
 	if storageConfig.RemoteCache {
```

Comment:
This is the part that needs to move into the pkg/meta 

pkg/meta dir has
<some-interface>.go
 - Interface defs
 - New(config)->struct-which-implements-interface

Reply:
@peusebiu storage should also move to this model.